server/routerlicious/packages/routerlicious-base/README.md

```diff
@@ -0,0 +1,3 @@
+# @fluidframework/server-routerlicious-base
+
+The Routerlicious Base package exports the fundamentals of Riddler and Alfred components that can be consumed and customized within a routerlicious implementation.
```

Comment:
I would not mention riddler and alfred in the readme. Looks too specific. Something like "base r11s constructs consumable by service providers" is probably better.

Reply:
r11s is too abstract, Though I now know it to be Routerlicious.
At present,
`The Routerlicious Base package exports base r11s constructs consumable by service providers.`
It's like just saying AB is BA   , A is Routerlicious  and B is Base.
And, ` consumable by service providers.` It would be nice to have an example.

Reply:
Please see the README in the subdirectory,
Might it be more friendly to people who read?